torch/_inductor/lowering.py

```diff
@@ -3651,6 +3651,16 @@ def op_add(a, b):
     return a + b
 
 
+@register_lowering(operator.sub)
+def op_sub(a, b):
+    return a - b
+
+
+@register_lowering(operator.truediv)
+def op_truediv(a, b):
+    return a / b
```

Comment:
Missing lowerings required to make things work. We have these on the symbolic shapes branch.